Modules/System/Language/src/LanguageImpl.Codeunit.al

```diff
@@ -149,6 +149,13 @@ codeunit 54 "Language Impl."
         if PAGE.RunModal(PAGE::"Windows Languages", WindowsLanguage) = ACTION::LookupOK then
             LanguageId := WindowsLanguage."Language ID";
     end;
+        
+    procedure GetParentLanguageId(LanguageId: Integer): Integer
+    var
+        CultureInfo: DotNet CultureInfo;
+    begin
+        exit(CultureInfo.CultureInfo(LanguageId).Parent.LCID);
```

Comment:
Do we need a null check here? Can the parent be null, leading to an exception when trying to get the LCID?

Reply:
It can also be a TryFunction and let the consumer decide

Reply:
I like the idea of the TryFunction. DotNet will throw an error but not one that is easy to read.

If I add a TryFunction, does that complicate the pull request because translations are required for the error message?

Reply:
Alternatively we can return the orriginal integer when the tryfunction fails.

Reply:
Implemented TryFunction. If there is no parent, it returns itsself.